pyproject.toml

```diff
@@ -12,8 +12,9 @@ packages = [
 
 [tool.poetry.dependencies]
 python = "^3.8"
-pysigma = "^0.11"
+pysigma = "^0.11 <=0.11.4"
 defusedxml = "^0.7.1"
+black = "^23.12"

```

Comment:
isn't black a dev dependency?

Reply:
it was :D 
but now it's a part of python backed, used to format generated code with current lint settings https://github.com/panther-labs/pySigma-backend-panther/blob/main/sigma/backends/panther/panther_backend.py#L4